assets/scripts/app/focus.js

```diff
@@ -44,18 +42,8 @@ export function onWindowFocus () {
   saveSettingsLocally()
 }
 
-function onWindowBlur () {
-  if (abortEverything) {
-    return
-  }
-
-  hideAllMenus()
-}
-
 function onVisibilityChange () {
-  if (document[system.visibilityState] === 'hidden') {
-    onWindowBlur()
```

Comment:
I go back and forth on this one.

The interesting thing I just realized is that the Page Visibility API is supported and no longer needs vendor prefixes on modern browsers. Theoretically, in the event listener added on line 7, the `visibilitychange` handler is the one that's being added on most browsers, not the `blur` event. (It's not one or the other, but either-or.)

For the sake of kicking this decision into the future, I'll re-add the `visibilitychange` handler to `menu.js` with a note.